homeassistant/components/min_max/sensor.py

```diff
@@ -72,28 +78,30 @@ async def async_setup_platform(hass, config, async_add_entities, discovery_info=
 def calc_min(sensor_values):
     """Calculate min value, honoring unknown states."""
     val = None
-    for sval in sensor_values:
+    entity = None
+    for entity_id, sval in sensor_values:
```

Comment:
Please rename `entity_id` to `sensor_id` to not overwrite the variable above. Rename `sval` to `sensor_value` while we're at it.